llvm/lib/Support/YAMLParser.cpp

```diff
@@ -392,6 +392,10 @@ class Scanner {
   ///        Pos is whitespace or a new line
   bool isBlankOrBreak(StringRef::iterator Position);
 
+  /// Return true if the minimal well-formed code unit subsequence at
+  ///        Pos is considered a "safe" character for plain scalars.
```

Comment:
```suggestion
  /// Position is considered a "safe" character for plain scalars.
```

Reply:
The declaration immediately before this also seems to have this unusual spacing:

```cpp
  /// Return true if the minimal well-formed code unit subsequence at
  ///        Pos is whitespace or a new line
  bool isBlankOrBreak(StringRef::iterator Position);
```

I also thought this was unusual, but I figured it would be better to be consistent with the existing code. That said, I am happy to change this if you still think it makes sense.

Reply:
You are right. Just keep your code for consistency.